src/image/jpeg_io.cpp

```diff
@@ -36,7 +36,7 @@ jpeg_error_exit (j_common_ptr cinfo)
 
 void parse_jpeg(const char* data, size_t length,
                 size_t& width, size_t& height,
-                size_t& channels) {
+                size_t& channels, const std::string& url) {
```

Comment:
Where is the `url` argument used?

Reply:
This is redundant - my bad. Thank you for the catch! I'll fix this.

image_util has access to url of images which it uses to log the error. So we don't need to include url as a parameter in parse_jpeg.